ansible/roles/prometheus/defaults/main.yml

```diff
@@ -43,5 +43,17 @@ prometheus_configuration:
       static_configs:
         - targets: ['localhost:9100']
 
-prometheus_rules:
-  groups: []
+prometheus_rules: |
```

Comment:
The default config has been wiped out, so this should go to lovelace's hostvars

Reply:
Done